tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -82,6 +83,24 @@ def create_parser(product_choices=None):
     mode_group.add_argument("--verify-log-full", action="store_true",
                             default=False,
                             help="Output per-iteration test results when running verify")
+    mode_group.add_argument("--verify-repeat-loop", action="store",
+                            default=10,
+                            help="Run tests in a loop for a number of times",
+                            type=int)
+    mode_group.add_argument("--verify-repeat-restart", action="store",
+                            default=5,
+                            help="Restart the whole process",
+                            type=int)
+    mode_group.add_argument("--verify-no-chaos-mode", action="store_false",
```

Comment:
I think this should use `dest="verify_chaos_mode"` since a True value means that chaos mode is used. Possibly there should also be a `--verify-chaos-mode` option so that you can specify a value even if we change the default.

Reply:
Let me see if I get this right you want me create a new --verify-chaos-mode and set both `--verify-chaos-mode` and `--verify-no-chaos-mode` with the same dest="verify_chaos_mode"? So that you can both enable and disable from the command line.

Reply:
Hey @jgraham could you confirm this? I'm not sure if that was what you meant. Thanks!

Reply:
Ah apologies, I missed the question. Yes, your understanding is perfect.